website/docs/r/appstream_image_builder.html.markdown

```diff
@@ -80,7 +80,7 @@ This resource exports the following attributes in addition to the arguments abov
 * `arn` - ARN of the appstream image builder.
 * `created_time` -  Date and time, in UTC and extended RFC 3339 format, when the image builder was created.
 * `id` - Name of the image builder.
-* `state` - State of the image builder. Can be: `PENDING`, `UPDATING_AGENT`, `RUNNING`, `STOPPING`, `STOPPED`, `REBOOTING`, `SNAPSHOTTING`, `DELETING`, `FAILED`, `UPDATING`, `PENDING_QUALIFICATION`
+* `state` - State of the image builder. Valid values: `PENDING`, `UPDATING_AGENT`, `RUNNING`, `STOPPING`, `STOPPED`, `REBOOTING`, `SNAPSHOTTING`, `DELETING`, `FAILED`, `UPDATING`, `PENDING_QUALIFICATION`
```

Comment:
Same suggestion as above, but this one does challenge my perception of this a bit. I added the `#state` anchor link tag, but it doesn't seem to really work, meaning this link goes to a fairly length page that you need to scroll down to the `state` header on.

In your opinion, does it make more sense to list here vs. linking? 

```suggestion
* `state` - State of the image builder. For valid values, see the [AWS documentation](https://docs.aws.amazon.com/appstream2/latest/APIReference/API_ImageBuilder.html#state)
```

Reply:
I was looking at the HTML of the documentation for the anchor and it seems that `#AppStream2-Type-ImageBuilder-State` works:

https://docs.aws.amazon.com/appstream2/latest/APIReference/API_ImageBuilder.html#AppStream2-Type-ImageBuilder-State

I'll update the PR with all the proposed changes with a new commit.